public/app/plugins/panel/barchart/bars.ts

```diff
@@ -226,7 +226,7 @@ export function getConfig(opts: BarsOptions, theme: GrafanaTheme2) {
 
       // Calculate final labels positions according to unified text size
       const textMeasurement = measureText(label.formattedValue, fontSize * devicePixelRatio);
-      const actualLineHeight = textMeasurement.fontBoundingBoxAscent + textMeasurement.fontBoundingBoxDescent;
+      const actualLineHeight = textMeasurement.actualBoundingBoxAscent + textMeasurement.actualBoundingBoxAscent;
```

Comment:
second one should be `..Descent`?

Reply:
Ah you're right. Dangers of autocomplete 😓